.editorconfig

```diff
@@ -120,7 +120,7 @@ dotnet_code_quality_unused_parameters = all:suggestion
 [*.cs]
 csharp_indent_labels = one_less_than_current
 csharp_using_directive_placement = outside_namespace:error
-csharp_prefer_simple_using_statement = true:error
```

Comment:
Can we keep this as an error?

Reply:
While the simplified using is nicer in 95%+ places, anywhere that we want to explicitly make sure the `.Dispose()` is called prior to continuing still needs the old syntax.
E.g., doing file operations, but needing the using's `.Dispose()` to close and flush the stream before opening a second time in the same method.